ci/kind/test-e2e-kind.sh

```diff
@@ -220,6 +220,9 @@ if $extra_network && [[ "$mode" == "hybrid" ]]; then
 fi
 
 if [[ $cleanup_only == "true" ]];then
+  if [[ $flexible_ipam == "true" ]]; then
+    $TESTBED_CMD destroy kind --flexible-ipam 
+  fi
```

Comment:
Ideally, the `kind-setup.sh` script would be smart enough so that passing the same flag (`--flexible-ipam`) when destroying the testbed is not required, as it is error prone (one can forget the flag and end up with an incomplete cleanup). However, this can be handled as a future improvement.